src/api/app/views/webui2/webui/staging_workflows/request_ignorations/index.html.haml

```diff
@@ -0,0 +1,33 @@
+.row
+  .col-xl-12
+    .card.mb-3
+      .card-body
+        %h3 Ignored Requests
```

Comment:
The route (and URL) is `request_ignorations`, but the view is displaying `Ignored Requests`. I think we should be consistent.

Reply:
Please don't call it Request Ignoration in the UI:-) I doubt anyone would understand what that would be.

Ideally I would also rename the association.

Reply:
Can any of us provide a "good" enough name for it? is the main problem we face here

Reply:
Things in the UI don't **have** to be named as things in the code. Those are two different contexts...